cli-toolchain/quickstart.md

```diff
@@ -572,7 +576,15 @@ As you can see in the prompt above, you can either choose to use an existing Lam
 
 #### DynamoDB Lambda Triggers
 
-You can associate a Lambda trigger with a DynamoDB table, managed by the Amplify CLI, using the amplify add/update storage flows. When attempting to add/update a DynamoDB storage resource, you would get the following CLI prompts to add a trigger for it.
+You can associate a Lambda trigger with a DynamoDB table, managed by the Amplify CLI. There are two ways by which DynamoDB is provisioned by the Amplify CLI
+
+- As a part of the Storage category
+- As a part of the GraphQL API (types with @model annotation)
```

Comment:
Can you hyperlink to the GraphQL Transformer?

Reply:
Done